### PR TITLE
4669 - Documentation for vuetify Lists updated

### PR DESCRIPTION
**Pull request by marsoloGit**
… updated, vuetify code example added, methods corrected/updated

### CODE REVIEW

**Comment by marsoloGit** [comment]
#[https://github.com/jdi-testing/jdi-light/issues/4669](url)

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -1548,47 +1548,243 @@ and [JDI vuetify page tests for windows](https://github.com/jdi-testing/jdi-ligh
...
+    @UI("#FlatList .v-list")
+    public static VuetifyList flatList;
+    
+    @Test(description = "Test checks if list is flat or not")
+    public void flatListsTest(){
+        flatList.show();
+        flatList.is().flat();
+        darkList.show();
+        darkList.is().notFlat();
+    }
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:1705]: If I am not mistaken we discussed that we take 1-2-3 tests, not all of them :)
And also master task says '2.	Element usage example (typically we use **one** of element's tests for that)'

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -1548,47 +1548,243 @@ and [JDI vuetify page tests for windows](https://github.com/jdi-testing/jdi-ligh
 
 ### 5.14 Lists
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/lists/)
```
Comment (vklonin) [inline source/includes/documentation/_vue_js_elements.md:1551]: Please consider to make part about Tests less extensive - find few examples that is most interesting and valuable from your pain of view.

Comment (marsoloGit) [reply source/includes/documentation/_vue_js_elements.md:1551]: Only two tests examples are left now.

File: source/includes/documentation/_vue_js_elements.md (hunk 1 above)
Comment (marsoloGit) [reply source/includes/documentation/_vue_js_elements.md]: Done. Only two tests examples are left now.

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (marsoloGit) [reply source/includes/documentation/_vue_js_elements.md:1551]: I decreased amount of tests examples

**Review by MayaElf — APPROVED** [review]

**Pull request merged**